### PR TITLE
Added a button admins can press to motivate mechanics to perform their duties

### PR DESCRIPTION
**Pull request by DamianX**
worn sprite by @BomberBro
item sprite by @SageOfLight

### CODE REVIEW

**Comment by DeityLink** [comment]
Work jobbie!

**Comment by Dacendeth** [comment]
Now we just need one to make xenoarchs and xenobiologists do research.

**Comment by hbycyl** [comment]
Our economic theory holds the human being is the most fundamental productive force. Except for those who must be exterminated physically out of political consideration, human beings must be utilized as productive forces, with submissiveness as the prerequisite. The Laogai system's fundamental policy is "Forced Labor as a means, while Thought Reform is our basic aim."

**Comment by ShiftyRail** [comment]
Needs changelog

**Comment by DamianX** [comment]
no it doesn't

File: code/modules/clothing/masks/necklaces.dm [hunk 1]
```diff
@@ -2,6 +2,130 @@
 	body_parts_covered = 0
 	ignore_flip = TRUE
 
+/obj/item/clothing/mask/explosive_collar
```
Comment (jknpj) [inline code/modules/clothing/masks/necklaces.dm:5]: Do you want mechanics to be able to scan their own death devices?

Comment (DamianX) [reply code/modules/clothing/masks/necklaces.dm:5]: It's not scannable

Comment (DamianX) [reply code/modules/clothing/masks/necklaces.dm:5]: also it's not a death device it's a cool necklace

**Comment by ShiftyRail** [comment]
No logs on button used ?

**Comment by DrSnips** [comment]
Since this adds a way determine if machines have been upgraded in a round. Would you mind adding an end of round statistic/counter if there isn't one already?

**Comment by GreatBigFailure** [comment]
Coders fear the mechanic.

**Comment by Zth--** [comment]
Would it explode on an electrical storm event?

**Comment by DamianX** [comment]
no

**Comment by SonixApache** [comment]
\>explosive collars
\>but only as admin button to fuck with mechanics
lotta effort for a shitpost

**Comment by jknpj** [comment]
3AI button is worse and it exists since the time of ye coders of olde.

**Comment by DeityLink** [comment]
Honestly I'd argue that we don't press it nearly enough.

**Comment by aceedex** [comment]
Admin button that makes stock parts cost plastic when

**Comment by AI-Is-Rogue** [comment]
Craftable bomb collars when? I want to enact my SAW fantasies.

**Comment by SonixApache** [comment]
Craftable collars when? I want to enact my fantasies.

**Pull request merged**